src/server/table.h

```diff
@@ -144,6 +144,13 @@ struct DbTable : boost::intrusive_ref_counter<DbTable, boost::thread_unsafe_coun
   size_t table_memory() const {
     return expire.mem_usage() + prime.mem_usage();
   }
+
+  // implement null-object pattern approach
+  SlotStats& GetSlotStats(SlotId slot);
+
+ private:
+  // without cluster we assume that we have only one slot
+  std::unique_ptr<SlotStats[]> slots_stats_;
```

Comment:
DbTable is struct therefore members should not end with _ and all fields are public